test/unit/content-info.test.js

```diff
@@ -23,6 +24,23 @@ describe('contentInfo module', function () {
         });
     });
 
+    // eslint-disable-next-line max-len
+    it('Should detect mp3 response stream if content-type is not present', function () {
+        // data url of mp3 file
+        // eslint-disable-next-line max-len
+        var data = fs.readFileSync('test/fixtures/audio.mp3'),
+            // creating the buffer of the file
+            response = new Response({ stream: Buffer.from(data, 'base64') });
```

Comment:
Isn't `data` already a  Buffer?

Reply:
I am now using a trimmed actual mp3 file not just the Base64 encoding of an mp3 in some file with a .mp3 extension. Reason: fs.readFileSync was reading the encoding as a text.

Reply:
`readFileSync` returns a buffer, this will also work:

```js
var response = new Response({
    stream: fs.readFileSync('test/fixtures/audio.mp3')
});
```